devops/wsi_deid/girder.local.conf

```diff
@@ -44,3 +44,4 @@ hide_metadata_keys_format_aperio = {
 hide_metadata_keys_format_hamamatsu = {
     "^internal;openslide;hamamatsu\.((AHEX|MHLN|YRNP|zCoarse|zFine)\[|(X|Y)OffsetFromSlideCentre|ccd.(width|height)|(focalplane|slant)\.(left|right)(top|bottom)|stage.center)": "^\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?)(\s*,\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?))*\s*$"}
 hide_metadata_keys_format_philips = {}
+import_text_association_column = 'LabelText'
```

Comment:
Ultimately (in a future PR, perhaps), we may want this to be a list of columns and find allow matching across columns.  For instance, you could have a name and a DOB on the label.

Reply:
I took a stab at this, see 78f345c. 

I'm thinking that it might require more robust logic than just 'if the OCR results match any of these columns...', although I'm not sure what that should look like.

Reply:
Maybe match any of the columns as you currently have, but if there are multiple matches and more than one column, you would then want to require that it matches at least two columns (and if multiple matches, then at least three, etc).  My mental model of this would be a column of names and a column of DOB (or first name, last name, DOB).  We expect the manifest to match the data, so only when there are multiple matches would we need to be more precise (except, of course, that the OCR is far from perfect, so this may ultimately not be the right thing).

Reply:
I changed match finding in 8f69f0d